app/services/bgs/payment_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module BGS
+  class PaymentService
+    include SentryLogging
+
+    def initialize(current_user)
+      @current_user = current_user
+    end
+
+    def payment_history
+      # rubocop:disable Rails/DynamicFindBy
+      response = service.payment_history.find_by_ssn(@current_user.ssn)
```

Comment:
Does the following not work?

```suggestion
      response = service.payment_history.find_by(ssn: @current_user.ssn)
```

Reply:
@f1337 Good question. This is not an ActiveRecord query. Rubocop thinks it is though, that's why I had to add the disable.

Here is what we're actually calling: https://github.com/department-of-veterans-affairs/bgs-ext/blob/8e994a59b2abe4204dde14147e33a5ac98a3e7c2/lib/bgs/services/payment_history.rb#L10